src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -117,9 +146,23 @@ export default {
 	computed: {
 		messageTime() {
 			return OC.Util.formatDate(this.timestamp * 1000, 'LT')
+		},
+		quote() {
+			if (this.parent) {
+				return this.$store.getters.message(this.token, this.parent)
+			} else return undefined
 		}
 	},
 	methods: {
+		handleReply() {
+			const MESSAGE_TO_BE_REPLIED = {
+				id: this.id,
+				actorDisplayName: this.actorDisplayName,
+				message: this.message,
+				token: this.token
+			}
+			this.$store.dispatch('addMessageToBeReplied', MESSAGE_TO_BE_REPLIED)
```

Comment:
Please extend to empty object or you'll have issues.
This will be passed as reference and every change to this object will impact all the other pushed `addMessageToBeReplied` :)

Reply:
@skjnldsv What do you mean here??

Reply:
MESSAGE_TO_BE_REPLIED is an object that will be passed as a reference.
Meaning any change of this object on the store can propagate the same changes into all the messages managed by the `addMessageToBeReplied` method.

You need to make sure you're sending a new object!
`this.$store.dispatch('addMessageToBeReplied', Object.assign({}, MESSAGE_TO_BE_REPLIED))`